### DISCUSSION TITLE
If the target test program requires two or more input files

### DISCUSSION THREAD

**Discussion by qwwq251** [AFLplusplus/AFLplusplus#1160] [Q&A]
If the target test program requires two or more input files, such as./target file1 file2, how should the fuzz procedure be constructed, whether the fuzz source code needs to be modified, or whether AFL/AFL ++ supports fuzz testing for the target? If yes, then whether the obtained crash is also a combination of two or more files.

**Comment by vanhauser-thc** [comment]
either you modify the program so it only requires one input, or one input is a fixed dummy file, e.g. like `afl-fuzz ... -- ./target @@ other_file.txt`
